src/content/chapter5_advanced_features/lesson04_externals/code.gleam

```diff
@@ -8,7 +8,7 @@ pub type DateTime
 pub fn now() -> DateTime
 
 // The `now` function in `./my_package_ffi.mjs` looks like this:
-// external function now() {
+// export function now() {
```

Comment:
I am not sure about this as I didn't try using external code. But from looking at the Lustre code (https://github.com/lustre-labs/lustre/blob/main/src/runtime.ffi.mjs), I assume that it is a typo.